recipes/ruby.rb

```diff
@@ -79,6 +84,7 @@
   end
 end
 
+before "deploy:update_code", "deploy:clean_old_dependencies"
```

Comment:
What would the benefit be? I'd rather explicitly run before `:update_code` in case the deploy task is over written in a subsequent files and causes unintended side effects.

Reply:
Ah, so the question I had when I read this was: "what comes before this stage that I might be concerned about?" Turns out it's nothing, and I can't see it being a problem anyway. Purely a suggestion to make the ordering a bit easier to understand. Happy to leave as-is.